VVResonances/python/tools/VectorBosonToolBox.py

```diff
@@ -94,10 +94,10 @@ def ZEEPFIsolation(self,z):
                         footPrintNeutralLeg1=footPrintNeutralLeg1+c.pt()
 
 
-        cleanedChargedIso1 = max(z.leg1.chargedHadronIsoR(0.4)-footPrintChargedLeg1,0.0)
-        cleanedChargedIso2 = max(z.leg1.chargedHadronIsoR(0.4)-footPrintChargedLeg2,0.0)
-        cleanedNeutralIsoRho1=max(z.leg1.neutralHadronIsoR(0.4)+z.leg1.photonIsoR(0.4)-footPrintNeutralLeg1-z.leg1.rho*z.leg1.EffectiveArea04,0)
-        cleanedNeutralIsoRho2=max(z.leg2.neutralHadronIsoR(0.4)+z.leg2.photonIsoR(0.4)--footPrintNeutralLeg2-z.leg2.rho*z.leg2.EffectiveArea04,0)
+        cleanedChargedIso1 = max(z.leg1.chargedHadronIso(0.4)-footPrintChargedLeg1,0.0)
+        cleanedChargedIso2 = max(z.leg1.chargedHadronIso(0.4)-footPrintChargedLeg2,0.0)
+        cleanedNeutralIsoRho1=max(z.leg1.neutralHadronIso(0.4)+z.leg1.photonIso(0.4)-footPrintNeutralLeg1-z.leg1.rho*z.leg1.EffectiveArea04,0)
+        cleanedNeutralIsoRho2=max(z.leg2.neutralHadronIso(0.4)+z.leg2.photonIso(0.4)--footPrintNeutralLeg2-z.leg2.rho*z.leg2.EffectiveArea04,0)
```

Comment:
@clelange @bachtis note that independently of this PR there's probably a typo in this line (the `--`)